gateway/main/urls.py

```diff
@@ -66,4 +63,5 @@
     re_path(r"^redoc/$", schema.with_ui("redoc", cache_timeout=0), name="schema-redoc"),
 ]
 if settings.DEBUG:
+    urlpatterns += [path("", include("django_prometheus.urls"))]
```

Comment:
I moved our prometheus url to DEBUG until we have time to do the integration with Grafana